packages/ui/primitives/document-flow/add-signers.tsx

```diff
@@ -38,9 +41,14 @@ export const AddSignersFormPartial = ({
   recipients,
   fields: _fields,
   onSubmit,
+  creatorEmail,
+  creatorName,
+  selfSign,
 }: AddSignersFormProps) => {
   const { toast } = useToast();
 
+  const [selfSignEnabled, setSelfSignEnabled] = useState<boolean>(selfSign);
```

Comment:
We can also change this to a memo and move it below the `useFieldArray` call.

Reply:
noted...making a commit.